src/pybind/mgr/pg_autoscaler/__init__.py

```diff
@@ -1 +1,6 @@
+import os
+
+if 'UNITTEST' in os.environ:
```

Comment:
@sebastian-philipp Thank you for your review, I've changed the style accordingly to that of the ceph-adm module